README.md

```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
 
 > If you did not receive a verification email, check your Spam or Junk email folders.
 
+#### Setting up your [Dolby.io](https://dashboard.dolby.io) app
+
+To set up your app for events, you will need to:
+
+1. Go to the _Dashboard_, and click `add new app` if you do not have an existing app. ![dashboard](events/documentation//dashboard-events.png)
+2. To enable events streaming, your app should be opted into the open beta program. You can find this at the bottom of the `Communications APIs` sidebar navigation when you click on your app.
+
+ ![dashboard](events/documentation/open-beta.png)
+
+## How to run the Events app
+
+Run the following steps after cloning the repository to run the application locally.
+
+### Install dependencies
+
+**note** : This guide is written with [Yarn](https://yarnpkg.com/) in mind. We have not validated this project with other package managers.
+
+Open a terminal window in the root directory of your project folder. Install the project's dependencies using the following command.
+
+```bash
+yarn
+```
+
+### Repo structure
+
+The code in this repository is organised in the following way
+
+* The `/events/src/` directory contains all the front-end code for the events app. Within this directory
+  * `hooks/` contains wrapper functions around our SDK for re-usable functionality.
+  * `components/` contains UI components that encapsulate and provide functionality.
+  * `utils/` provides some generic helper functions.
+  * `context/` contains the React Context for the side drawer and the main component window.
+
+* The `backend/` contains the code for the proxy server.
+
+This project is built with the [Comms UI Kit for react](https://github.com/dolbyio/comms-uikit-react) library for simplicity and re-use of standard Communications API based components.
+
+### Set up `.env` file
+
+Create a new file called `.env` in the `events` folder, and copy the contents of `.env.example` into it.
+
+You will need to provide the values for `KEY` and `SECRET` from your dolby.io app. To obtain your key and secret from the Dolby.io dashboard,
+
+1. Go to the _Dashboard_, and click on the `API Keys` next to your application.
+   ![dashboard](events/documentation/dashboard.png)
+2. On the next screen, copy the `App key` and `App secret` and paste them in your `.env` file against the marked variables.
+   ![token](events/documentation/client_access_token.png)
+
+If you want chat and the ability to promote a viewer to host, you will need to provide your `PUBNUB_PUBLISH_KEY`, `PUBNUB_SUBSCRIBE_KEY`, and `PUBNUB_SECRET_KEY` as well. To setup PubNub, please go through the [enabling chat guide in our customization guide](customization.md/#enabling-chat-and-promoting-a-viewer-to-co-host).
+
+The secrets will not be exposed on the frontend.
+
+#### Start the app
+
+Execute the following command to run the application locally in a terminal window inside the `/events/` directory.
```

Comment:
```suggestion
#### Running the app

To run the application locally, execute the command below inside the `/events` directory.
```

Also, we should change all of the headings to either use gerund (-ing words) or not (but not both)